sysutils/coreutils-uutils/Portfile

```diff
@@ -0,0 +1,249 @@
+    vec_map                                              0.8.2  f1bddf1187be692e79c5ffeab891132dfb0f236ed36a43c7ed39f1165ee20191 \
+    version_check                                        0.9.3  5fecdca9a5291cc2b8dcf7dc02453fee791a280f3743cb0905f8822ae463b3fe \
+    void                                                 1.0.2  6a02e4885ed3bc0f2de90ea6dd45ebcbb66dacffe03547fadbb0eeae2770887d \
+    walkdir                                              2.3.2  808cf2735cd4b6866113f648b791c6adc5714537bc222d9347bb203386ffda56 \
+    wasi                          0.9.0+wasi-snapshot-preview1  cccddf32554fecc6acb585f82a32a72e28b48f8c4c1883ddfeeeaa96f7d8e519 \
+    wasi                         0.10.2+wasi-snapshot-preview1  fd6fbd9a79829dd1ad0cc20627bf1ed606756a7f77edff7b66b7064f9cb327c6 \
+    wild                                                 2.0.4  035793abb854745033f01a07647a79831eba29ec0be377205f2a25b0aa830020 \
+    winapi                                               0.2.8  167dc9d6949a9b857f3451275e911c3f44255842c1f7a76f33c55103a909087a \
+    winapi                                               0.3.9  5c839a674fcd7a98952e593242ea400abe93992746761e38641405d28b00f419 \
+    winapi-build                                         0.1.1  2d315eee3b34aca4797b2da6b13ed88266e6d612562a0c46390af8299fc699bc \
+    winapi-i686-pc-windows-gnu                           0.4.0  ac3b87c63620426dd9b991e5ce0329eff545bccbbb34f3be09ff6fb6ab51b7b6 \
+    winapi-util                                          0.1.5  70ec6ce85bb158151cae5e5c87f95a8e97d2c0c4b001223f33a334e3ce5de178 \
+    winapi-x86_64-pc-windows-gnu                         0.4.0  712e227841d057c1ee1cd2fb22fa7e5a5461ae8e48fa2ca79ec42cfc1931183f \
+    xattr                                                0.2.2  244c3741f4240ef46274860397c7c74e50eb23624996930e484c16679633a54c
+
+checksums-append    ${distname}${extract.suffix} \
+                    rmd160  c5a028c84e0d45b8761bbee4ead64c306de55d37 \
+                    sha256  066359e9548940ee99c3d8911e951d5091aa1f2d7d409cb577c811f3993a1e7d \
+                    size    1783965
+
+build.pre_args-append --features macos
+
+set binaries {
+    arch base32 base64 basename cat chgrp chmod chown chroot cksum comm cp csplit
+    cut date df dircolors dirname du echo env expand expr factor false fmt fold
+    groups hashsum head hostid hostname id install join kill link ln logname ls
+    md5sum mkdir mkfifo mknod mktemp more mv nice nl nohup nproc numfmt od paste
+    pathchk pinky printenv printf ptx pwd readlink realpath relpath rm rmdir seq
+    sha1sum sha224sum sha256sum sha3-224sum sha3-256sum sha3-384sum sha3-512sum
+    sha384sum sha3sum sha512sum shake128sum shake256sum shred shuf sleep sort split
+    stat stdbuf sum sync tac tail tee test timeout touch tr true truncate tsort
+    tty uname unexpand uniq unlink uptime users wc who whoami yes
+}
```

Comment:
I am still not completely happy with a hardcoded list. Why cannot you glob the destroy area to automatically discover the provided utilities ?

Reply:
I think as was mentioned earlier, the binary operates busybox style, meaning that a single binary gets built named `coreutils`, and operates differently depending on what it is named or symlinked as.

Basically there is no source to glob.  This is a list of symlinks to all create pointing to `coreutils`, and when `coreutils` gets called through any one of these links, it behaves as the relevantly-named utility.

Reply:
I understood that. I also understood there was a way to build differently that wasn't this way, and man point was for me that would be preferred.

Reply:
If it's trivial to do, then I agree. If not, then I'm fine with the hard-coded list.

Anyone else?

Reply:
Chris, given that Dave has been very willing to make changes/improvements - and in the interest of not discouraging new port maintainers - I'm going ahead and merging this. Two of us have approved, and overall everything looks good.

If you still feel strongly regarding the hard-coded list, go ahead and file a ticket. Since Dave is also willing to maintain this port, he can take a look at your idea when he has time.